client/actions.js

```diff
@@ -34,6 +34,7 @@ export const ADMIN_SET_PERMISSIONS = 'ADMIN_SET_PERMISSIONS'
 // The AudioManager has finished initializing and is ready to play sounds
 export const AUDIO_MANAGER_INITIALIZED = 'AUDIO_MANAGER_INITIALIZED'
 
+export const AUTH_ACCOUNT_UPDATE = 'AUTH_ACCOUNT_UPDATE'
```

Comment:
Yeah, since we're using ID-ed requests for auth actions, we have a single action `AUTH_CHANGE_BEGIN` that is used there:
https://github.com/ShieldBattery/ShieldBattery/blob/master/client/auth/auther.js#L16

However, I'm not entirely sure if that should be applicable here?

Reply:
Feels a bit odd to me, but it's probably fine. We do use it in password reset too, so I guess using it here makes sense.